src/BlockchainReader/Bitcoin.ts

```diff
@@ -29,35 +29,41 @@ const transactionToOutputs = (transaction: Transaction): ReadonlyArray<VOutWithT
 
 const outputIsDataOutput = (output: VOut) => output.scriptPubKey.type === 'nulldata'
 
-const dataOutputToPoetAnchor = (dataOutput: VOutWithTxId): TransactionPoetTimestamp => {
-  // TODO: split method in two. see https://github.com/poetapp/node/issues/418
+const dataOutputToPoetTransactionAnchor = (output: VOutWithTxId): PoetTransactionAnchor => {
+  const anchor = dataToPoetAnchor(dataOutputToData(output))
+  return {
+    ...anchor,
+    transactionId: output.transactionId,
+  }
+}
+const dataOutputToData = (dataOutput: VOutWithTxId): string => {
```

Comment:
can you add an empty line?

Reply:
Thanks! Will do. 

I'll wait until https://github.com/poetapp/node/pull/470 is merged so the diff on this PR becomes much easier to read.